common/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+add_library(umd_common INTERFACE)
```

Comment:
@joelsmithTT FYI, related to #290 , you can rebase after this change, and it will be easier for the new logging library

Reply:
ok, I will wait for this to go in, then rebase & merge #290.